dashboard/test/controllers/api/v1/sections_controller_test.rb

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+require 'test_helper'
+
+class Api::V1::SectionsControllerTest < ActionController::TestCase
+  self.use_transactional_test_case = true
```

Comment:
Puts every test case in its own SQL transaction, helping to isolate tests. See [transactional_test_case.rb](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/testing/transactional_test_case.rb).  I think this is fairly important to the `setup_all` step in this file.